### ISSUE TITLE
property:typeElement

### ISSUE THREAD

**Issue by nielsklazenga** [tdwg/tcs2#212]
### typeElement (property)

|    |    |
|----|----|
| Label | Type Element |
| Definition | Element by which a taxon name is typified.  |
| Usage notes | |
| Comments | Names of species and infraspecific taxa are mostly typified by a specimen. One could use a Darwin Core Preserved Specimen object for that. Names of higher taxa are mostly typified by the name of a species or genus. A TCS Taxon Name should be used for that. |
| Required | Yes |
| Repeatable | No |
| Constraints | Object or IRI |

**Comment by nielsklazenga** [comment]
I propose to have this property instead of `typeSpecimen` (#62) and `typeName` (#61). For names of species and infraspecific taxa the object can then be a Darwin Core Preserved Specimen or an IRI that points to any type of specimen record and for names of higher taxa it can be (or should be) as TCS Taxon Name. I think this approach has a number of advantages:

- We do not have to use different properties for typified names of different ranks. Ultimately names of all ranks are typified by specimens (it could be the type specimen of the type species or of the type species of the type genus).
- We can deal with names that are typified by published illustrations or photos (I think that is still allowed in algae) without having to change the standard.
- We can make the property required for Nomenclatural Type objects.

**Comment by jliljeblad** [comment]
>     * We can deal with names that are typified by published illustrations or photos (see [property:typeImage #64](https://github.com/tdwg/tcs2/issues/64)) without having to change the standard.

Good idea, I think there was a (accepted) case with a fly being described from a photo since it was never actually caught. Also, who knows what will happen with typification by some kind of molecular sequence.
